src/sqlx_store.rs

```diff
@@ -14,15 +14,18 @@ use sqlx::{sqlite::SqliteRow, SqlitePool};
 use crate::{user_store::UserStore, AuthUser};
 
 const TABLE_NAME_TEMPLATE: &str = "{{table_name}}";
+const COLUMN_NAME_TEMPLATE: &str = "{{column_name}}";
 
 /// A generic SQLx user store.
 ///
-/// Concrete implementions are provided as well and should usually be used
+/// Concrete implementations are provided as well and should usually be used
 /// unless generics are required by the application.
 #[derive(Clone, Debug)]
 pub struct SqlxStore<Pool, User, Role = ()> {
     pool: Pool,
     table_name: String,
+    column_name: String,
+    query: Option<String>,
```

Comment:
@maxcountryman 
Could you clarify what that would look like?
Remove `table_name` and `column_name`  fields and jest keep `query` with a reasonable default (`select * from users where id = $1`)?

Reply:
Exactly. It's a breaking change technically, but I'm questioning the wisdom of specifying table and column names instead of the whole query itself (I should have thought about this more thoroughly from the outset, but here we are).